backend/start_server.py

```
@@ -51,10 +56,25 @@ def check_model():
     """Start the API Server"""
     print("Starting Smells Like Job Spirit API server...")
-    subprocess.run([sys.executable, "-m", "uvicorn", "app.main:app", "--reload", "--host", "0.0.0.0", "--port", "8000"])
+    subprocess.run(
+        [
+            sys.executable,
+            "-m",
+            "uvicorn",
+            "app.main:app",
+            "--reload",
+            "--host",
+            "0.0.0.0",
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)